deepmd/pt/utils/tabulate.py

```
@@ -0,0 +1,920 @@
+        elif activation_fn == ACTIVATION_FN_DICT["sigmoid"]:
+            self.functype = 6
+        else:
+            raise RuntimeError("Unknown activation function type!")
+        self.activation_fn = activation_fn
+
+        if isinstance(self.descrpt, deepmd.pt.model.descriptor.DescrptSeR) or isinstance(
+                self.descrpt, deepmd.pt.model.descriptor.DescrptSeA) or isinstance(
+                self.descrpt, deepmd.pt.model.descriptor.DescrptSeT) or isinstance(
+                self.descrpt, deepmd.pt.model.descriptor.DescrptSeT) or isinstance(
+                self.descrpt, deepmd.pt.model.descriptor.DescrptDPA1
+            ):
```

Ruff

Multiple `isinstance` calls for expression, merge into a single call

Merge `isinstance` calls

(SIM101)